Spatial.gd

```diff
@@ -849,14 +855,19 @@ func new_trail_point_after(waypoint_trail: Waypoint.Trail, trail3d: Spatial, tra
 
 func refresh_trail3d_points(trail3d: Spatial):
 	trail3d.refresh_mesh()
+	set_unsaved_changes(true)
 
 func refresh_trail2d_points(trail2d: Line2D):
 	trail2d.refresh_points()
+	set_unsaved_changes(true)
 
 
 ################################################################################
 # Signal Functions
 ################################################################################
+func _on_SaveTrail_pressed():
+	save_current_map_data()
+
```

Comment:
Isn't this button supposed to take data from the split files and save it to the combined files, or in the current temporary case: "the one big combined file which we will eventually split out into individual combined files"